swag_client/schemas/v1.py

```diff
@@ -45,9 +46,9 @@ class AccountSchema(Schema):
     # Set to False if this is a partner account your apps may need to know about.
     ours = fields.Boolean(required=True)
 
-    schema_version = fields.Integer(required=True, missing='v1')
+    schema_version = fields.Integer(missing='v1')
     account_status = fields.String(missing='created')
 
     @validates_schema
-    def validate_metadata(self, data):
-        TYPES[data['type']](many=True, strict=True).load([data['metadata']])
+    def validate_metadata(self, data, partial=False, many=False, unknown=False):
+        TYPES[data['type']](many=True).load([data['metadata']])
```

Comment:
`many=False` in the function signature, but `many=True` is passed in here. Not a dealbreaker, but just curious.

Reply:
many=True is to handle a collection of objects, it was set previously so I just left it as-is